src/Command/ResetProgrammaticFoundationCommand.php

```diff
@@ -0,0 +1,54 @@
+<?php
+
+namespace AppBundle\Command;
+
+use AppBundle\Entity\ProgrammaticFoundation\Approach;
+use Doctrine\ORM\EntityManagerInterface;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Console\Style\SymfonyStyle;
+
+class ResetProgrammaticFoundationCommand extends Command
```

Comment:
what about adding an option on the first command, like `doctrine:fixture:load --apend`
>--append               Append the data fixtures instead of deleting all data from the database first.

Reply:
fixtures do not exist in production !